integration-tests/tests/otlp-metrics/otlp_metrics_test.go

```diff
@@ -1,23 +1,80 @@
 package main
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/grafana/agent/integration-tests/common"
 	"github.com/stretchr/testify/assert"
 )
 
-const query = "http://localhost:9009/prometheus/api/v1/query?query=span_metrics_duration_bucket{test_name='otlp_metrics'}"
+const promURL = "http://localhost:9009/prometheus/api/v1/query?query="
 
-func TestOtlpMetrics(t *testing.T) {
+func metricQuery(metricName string) string {
+	return fmt.Sprintf("%s%s{test_name='otlp_metrics'}", promURL, metricName)
+}
+
+func TestOTLPMetrics(t *testing.T) {
+	tests := []struct {
+		query  string
+		metric string
+	}{
+		// TODO: better differentiate these metric types?
+		{metricQuery("example_counter"), "example_counter"},
```

Comment:
Could we remove duplicated strings here and in other places? I think you can find a way to only write "example_counter" etc only once per line.